### PR TITLE
custom-select-box

### PR DESCRIPTION
**Pull request by mujeeb100**

### CODE REVIEW

**Review by faran4engg — CHANGES_REQUESTED** [review]
correct folder name 😒 here too

**Review by coder-mezab — APPROVED** [review]

File: tasks-by-users/muzeeb100/Custom Select Box css/style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    background: #0563af;
+    color: #fff;
+    padding: 10px;
+    width: 250px;
+    height: 50px;
+    border: none;
+    font-size: 20px;
+    box-shadow: 0 5px 25px rgba(0,0,0,.5);
+    -webkit-appearance: button;
+    outline: none;
```
Comment (AbbasShaikh0304) [inline tasks-by-users/muzeeb100/Custom Select Box css/style.css:25]:
```suggestion
    background: #0563af;
    color: #fff;

    width: 250px;
    height: 50px;
    padding: 10px;

    font-size: 20px;

    border: none;
    box-shadow: 0 5px 25px rgba(0,0,0,.5);
    outline: none;

    -webkit-appearance: button;
```

**Review by AbbasShaikh0304 — CHANGES_REQUESTED** [review]

File: tasks-by-users/muzeeb100/Custom Select Box css/style.css (hunk 1 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/muzeeb100/Custom Select Box css/style.css:25]: Group Similar Properties so it will look more clear

Comment (AbbasShaikh0304) [reply tasks-by-users/muzeeb100/Custom Select Box css/style.css:25]: ????

**Pull request merged**